Content.Server/_Sunrise/AddWantedStatus/AddWantedStatusSystem.cs

```diff
@@ -0,0 +1,111 @@
+using Content.Server.CriminalRecords.Systems;
+using Content.Server.Radio.EntitySystems;
+using Content.Server.Station.Systems;
+using Content.Server.StationRecords.Systems;
+using Content.Shared._Sunrise.AddWantedStatus;
+using Content.Shared.Actions;
+using Content.Shared.Humanoid;
+using Content.Shared.IdentityManagement;
+using Content.Shared.IdentityManagement.Components;
+using Content.Shared.Security;
+using Content.Shared.StationRecords;
+using Content.Shared.Inventory;
+
+namespace Content.Server._Sunrise.AddWantedStatus;
+
+public sealed partial class AddWantedStatusSystem : EntitySystem
+{
+    [Dependency] private readonly SharedActionsSystem _actions = default!;
```

Comment:
```suggestion
    [Dependency] private readonly ActionsSystem _actions = default!;
```

На серверных система предпочтительно использовать серверные реализации систем